tasks/api/src/main/java/org/projectnessie/nessie/tasks/api/TaskState.java

```diff
@@ -20,6 +20,7 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.base.Supplier;
```

Comment:
Java `Supplier`?

Reply:
removed this from current PR... will follow-up separately.